### PR TITLE
Add a postgres image

### PR DESCRIPTION
**Pull request by thomaseizinger**
This PR adds support for a postgres image.

### CODE REVIEW

File: images/postgres/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[package]
+name = "tc_postgres"
+version = "0.1.0"
+authors = [ "CoBloX developers <team@coblox.tech>" ]
+description = "Testcontainers image for the parity docker image."
```
Comment (thomaseizinger) [inline images/postgres/Cargo.toml]:
```suggestion
description = "Testcontainers image for the postgres docker image."
```

File: images/postgres/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[package]
+name = "tc_postgres"
+version = "0.1.0"
+authors = [ "CoBloX developers <team@coblox.tech>" ]
```
Comment (D4nte) [inline images/postgres/Cargo.toml]: Not sure this is valid as we don't use postgres (do we ?)

Comment (thomaseizinger) [reply images/postgres/Cargo.toml]: Well the field doesn't say `users` but `authors` :)

Unless what you want to say is: I don't want to maintain this!

Comment (D4nte) [reply images/postgres/Cargo.toml]: I am saying I don't think CoBloX should maintain this.

Comment (thomaseizinger) [reply images/postgres/Cargo.toml]: Fair enough. I'll add my name then :)

Comment (thomaseizinger) [reply images/postgres/Cargo.toml]: Done.

File: Cargo.toml [hunk 3]
```diff
@@ -6,6 +6,8 @@ members = [
     "images/dynamodb_local",
     "images/elasticmq",
     "images/parity_parity",
+    "images/postgres",
+    "images/trufflesuite_ganachecli",
```
Comment (D4nte) [inline Cargo.toml]:
```suggestion

```

**Pull request merged**